### PR TITLE
Removing Everflow acl table along with DATAACL to free hw resources for test_null_route_helper.py

### PR DESCRIPTION
**Pull request by kbabujp**
### Description of PR
test_null_route_helper.py deleting only DATAACL before starting the test and adding 2 new ACL tables in the test case.
If there is no enough hardware resource, the second table creation fails. For successful creation of 2 new acl tables, deleting one more acl table to free-up space.

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [*] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205
- [*] 202305

### CODE REVIEW

File: tests/acl/null_route/test_null_route_helper.py [hunk 1]
```diff
@@ -68,35 +69,48 @@
...
-        entry = json.loads(output)[TABLE_NAME]
-        cmd_create_table = "config acl add table {} {} -p {} -s {}"\
-            .format(TABLE_NAME, entry['type'], ",".join(entry['ports']), entry['stage'])
-        logger.info("Restoring ACL table {}".format(TABLE_NAME))
-        rand_selected_dut.shell(cmd_create_table)
-    except Exception as e:
-        pytest.fail(str(e))
+    # Recover DUT by reloading minigraph
+    for duthost in duthosts:
+        config_reload(duthost, config_source="minigraph")
```
Comment (wangxin) [inline tests/acl/null_route/test_null_route_helper.py:113]: 1. Config reload takes time. Is there a faster way to restore the removed ACL tables?
2. If config reload is the only way to restore configuration, please use `config_source="running_golden_config"`.

Comment (kbabujp) [reply tests/acl/null_route/test_null_route_helper.py]: Thanks for your comments, i have removed config reload and used other method to reconfigure the acl.

File: tests/acl/null_route/test_null_route_helper.py [hunk 2]
```diff
@@ -68,35 +68,59 @@
...
+        rand_selected_dut.shell(cmd_create_table)
+
+    if TABLE_NAME_2 in entry_json:
+        entry = json.loads(output)[TABLE_NAME_2]
         cmd_create_table = "config acl add table {} {} -p {} -s {}"\
-            .format(TABLE_NAME, entry['type'], ",".join(entry['ports']), entry['stage'])
-        logger.info("Restoring ACL table {}".format(TABLE_NAME))
+            .format(TABLE_NAME_2, entry['type'], ",".join(entry['ports']), entry['stage'])
+        logger.info("Restoring ACL table {}".format(TABLE_NAME_2))
         rand_selected_dut.shell(cmd_create_table)
```
Comment (bingwang-ms) [inline tests/acl/null_route/test_null_route_helper.py:123]: The logic looks good to me. It should be better if you put `TABLE_NAME_1` and `TABLE_NAME_2` in a list and walk through them in a loop.

Comment (kbabujp) [reply tests/acl/null_route/test_null_route_helper.py]: Addressed the comments. Please check.

**Review by wangxin — APPROVED** [review]

**Pull request merged**